src/locales/de/framework.ftl

```diff
@@ -3,3 +3,37 @@
 ### among different targets.
 
 framework-validation-required = Dies ist ein Pflichtpfeld.
+
+framework-timeago =
```

Comment:
These German Translations are in Spanish?

Reply:
These are for you haha. I'm really rusty with german.

Reply:
Haha sure